### PR TITLE
Fix print in advance button copy

### PR DESCRIPTION
**Pull request by cibelius**
## Description

Flagged by Alta: https://opencrvsworkspace.slack.com/archives/C02P91N0E1W/p1782380441539319

## Checklist

- [ ] I have linked the correct Github issue under "Development"
- [ ] I have tested the changes locally, and written appropriate tests
- [ ] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [ ] I have updated the changelog with this change (if applicable)
- [ ] I have updated the GitHub issue status accordingly

### CODE REVIEW

File: src/events/birth/forms/declaration.ts [hunk 1]
```diff
@@ -29,14 +39,14 @@ export const BIRTH_DECLARATION_REVIEW = {
...
         id: 'event.birth.action.declare.form.review.print.label',
         description: 'Label for the print button in the review section'
       },
       configuration: {
         template: 'v2.birth-certified-certificate',
         buttonLabel: {
-          defaultMessage: 'Print certificate inadvance of registration',
+          defaultMessage: 'Print certificate in advance of registration',
           description: "Print button's label",
           id: 'event.birth.action.declare.form.review.print.button.label'
```
Comment (Nil20) [inline src/events/birth/forms/declaration.ts]: maybe change the id to match `event.birth.action.declare.form.review.print.label`
```suggestion
          id: 'event.birth.action.declare.form.review.print.label'
```

**Review by Nil20 — APPROVED** [review]
